tests/unit/LocaleCompareTest.js

```diff
@@ -0,0 +1,27 @@
+const {default: localeCompare} = require('@libs/LocaleCompare');
+
+describe('localeCompare', () => {
```

Comment:
I have asked to add more tests to verify that the Spanish locale works asa expected/ compared to English